backend/config.py

```diff
@@ -101,6 +102,39 @@ def from_dict(**kwargs):
         return ofx_import
 
 
+class Rule:
+
+    class MatchingOperator(enum.Enum):
+        IS = 'is',
+        IS_NOT = 'is not'
+        CONTAINS = 'contains',
+        STARTS_WITH = 'starts with',
+        ENDS_WITH = 'ends with'
+
+    # payee: str
+    matching_operator: MatchingOperator
+    matching_text: str
+    category_id: int
+
+    @staticmethod
+    def from_dict(**kwargs):
+        rule: Rule = Rule()
+        type_hints: Dict = typing.get_type_hints(Rule)
```

Comment:
This is awesome that you can get a Python class' field type hints :smile: :+1: !!